docs/getting-started/integrate-auth/react-native.mdx

```diff
@@ -0,0 +1,161 @@
+---
+id: react-native
+title: Integrate Authentication into React Native
+sidebar_label: React Native
+---
+
+import CodeFromRemote from "@theme/CodeFromRemote"
```

Comment:
Please wrap all MDX code in an [mdx code block](https://github.com/ory/docs/#import-markdown) in this document!